slides/intro-prubeh.md

```diff
@@ -9,7 +9,7 @@
 
 # 2. Zkusíte si
 
-- Zadání se objeví v aplikaci.
+- Zadání se objeví v aplikaci: czchts.link/web
```

Comment:
Poslední informace, kterou mám, je, že Czechitas bude používat svůj zkracovač via czechitas.cz a tenhle se nechá umřít.

Reply:
Tak až tak nastane, tak to vyměníme :-)

Reply:
Asi souhlas, ale díky @katemihalikova za varování.